ci/gcov_html/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
```

Comment:
this file and 'sorry.gif' are unnecessary

Reply:
No. if administrator does not set-up gcov/lcov, it is available.